_posts/2020/10/2020-10-28-release-notes.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+date: 2020-10-28
+title: "0.12: Release Notes"
+author: olenagerasimova
+description: |
+  Key features and improvements description in the recent
+  version 0.12, released just a few hours ago.
+keywords:
+  - artipie
+  - binary artifacts
+  - hosting of artifacts
+  - maven
+  - artipie rest api
+  - users
+  - permissions
+---
+
+v0.12 was released on NN. In October, we focused on extending and adding proxy for our adapters. 
```

Comment:
@olenagerasimova I do not think that we should be using word `adapter` in release notes. `adapter` is word like module, it is for people aware of Artipie's internals. Release notes are targeted at users and for them term `repository` or `repository type` would be more familiar. So I'd suggest to change wording here and in other parts of release notes

Reply:
@olegmoz hm, I do not think "repository" is a suitable word here as "repository" is just a specifically organized set of files, and I'm talking about a library (or a module) here. Also, we've already used word "adapter" in the previous release notes, so I think it's ok to use it farther.

Reply:
@olenagerasimova ok, I see. It's okey if we are using word `adapter` already